spyder/plugins/variableexplorer/widgets/main_widget.py

```diff
@@ -420,18 +420,24 @@ def setup(self):
 
     def update_actions(self):
         """Update the actions."""
+        nsb = self.current_widget()
+        if self.is_current_widget_empty():
+            return
+
```

Comment:
Similar as the comment above, maybe this validation could go before the `nsb = self.current_widget()` line?